app/models/sample.rb

```diff
@@ -481,6 +481,10 @@ def reload_old_pipeline_runs
     end
   end
 
+  def self.pipeline_commit(branch)
```

Comment:
maybe this would make more sense as a PipelineRun function?

Reply:
yeah... since it's just a global function, i wish we could put it outside of any ActiveRecord class